spec/50-privacy.md

```diff
@@ -21,7 +21,3 @@ The `tracestate` field may contain any <a href="#opaque">opaque</a> value in any
 Vendors MUST NOT include any personally identifiable information in the `tracestate` header.
 
 Vendors extremely sensitive to personal information exposure MAY implement selective removal of values corresponding to the unknown keys. Vendors SHOULD NOT mutate the `tracestate` field, as it defeats the purpose of allowing multiple tracing systems to collaborate.
-
-## Other risks
```

Comment:
I see this section as section 6.3 in https://www.w3.org/TR/trace-context/ (W3C Recommendation 23 November 2021 version) - is that an unintended inclusion that we must remove (since the response headers are not part of the recommendation yet), or am I missing anything?

Reply:
I'm not sure. I removed it because it seems pretty clearly to be talking about response headers, but if it was already in there is probably no risk in keeping it.

Reply:
OK, thanks. I will add this for discussion in the next W3C DT working group meeting (on whether we should continue having this text in the recommendation stage document or whether we should consider removing it from there since response headers don't apply there).

Reply:
Filed issue https://github.com/w3c/trace-context/issues/487 for this per our discussion in the WG meeting.